deploy/010-deploy-pools.ts

```diff
@@ -52,22 +52,28 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   const { tokenConfig, poolConfig } = await getConfig(hre.network.name);
 
   for (let i = 0; i < poolConfig.length; i++) {
+    let pools = await poolRegistry.callStatic.getAllPools();
```

Comment:
I would say we can do this out of the `for` loop, because we need to update it only if we create a new pool (as we are doing in the `if` statement)

Reply:
yeah we could do that